Assets/Scripts/NewBulletBehaviour.cs

```diff
@@ -57,7 +57,31 @@ private void Update()
 
         private void OnTriggerEnter(Collider other)
         {
-            ApplyDamage(other.transform);
+            if (_bulletConfig.activateAOE)
+            {
+                Vector3 pos = new Vector3(transform.position.x, 2.5f, transform.position.z);
+                Ray ray = new Ray(pos, Vector3.down);
+                var hitall = Physics.SphereCastAll(ray, _bulletConfig.hitAreaRadius);
+                foreach (var hit in hitall)
+                {
+                    ApplyDamage(hit.transform);
+                }
+                
+                // Debuging
+                if (Debugger.ShowAreaEffectVisualizer)
```

Comment:
Side note: if possible, can we display the visual effects without the debugger? Would it cause a perf hit? I added https://app.clickup.com/t/8696311g7 with lower priority to track this, if you have cycles for this